Making Sense/Models/CustomerCRUD.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Making_Sense.Interface.ICarCrud;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+
+namespace Making_Sence.Models
+{
+    public class CustomerCRUD : ICRUD<Customer>
+    {
+        private readonly string jsonFileCustomer = ConfigurationManager.AppSettings["pathFileCustomer"];
+        public List<Customer> customerList = new List<Customer>();
+        public Customer Create(Customer customer)
+        {
+            customerList.Add(customer);
+            Save();
+            return customer;
+        }
+        private void Save()
+        {
+            string customerJson = JsonConvert.SerializeObject(customerList.ToArray(), Formatting.Indented);
+            File.WriteAllText(jsonFileCustomer, customerJson);
+        }
+        public Customer Get(int id)
+        {
+            List<Customer> customers = JsonConvert.DeserializeObject<List<Customer>>(Read());
+            var aux = new Customer();
```

Comment:
Se podrá utilizar un nombre mas representativo? 
```suggestion
            var customer = new Customer();
```